doc-content/enterprise-only/processes/process-instance-migration-creating-plan-proc.adoc

```diff
@@ -0,0 +1,40 @@
+[id='process-instance-migration-creating-plan-proc']
+= Creating a migration plan
+
+You can define the migration between two different process definitions, known as a migration plan, in the process instance migration (PIM) service web UI.
+
+.Prerequisites
+* You have defined processes in a backup-ed {PRODUCT} development environment.
+* The process instance migration service is running.
+
+.Procedure
+. Enter `http://localhost:3300` in a web browser.
+. Log in to the PIM service.
+. In the upper right corner of the *Process Instance Migration* page, from the KIE Service list select the KIE Service you want to add a migration plan for.
+. Click *Add Plan*. The *Add Migration Plan Wizard* window opens.
+. In the *Name* field, enter a name for the migration plan.
+. Optional: In the *Description* field, enter a description for the migration plan.
+. Click *Next*.
+. In the *Source ContainerID* field, enter the source container ID.
+. In the *Source ProcessId* field, enter the source process ID.
+. Click *Copy Source To Target*.
+. In the *Target ContainerID* field, update the target container ID.
+. Click *Retrieve Definition from backend* and click *Next*.
++
+image::processes/pim-migration-wizard.png[align="center"]
+
+. From the *Source Nodes* list, select the source node you want to map.
+. From the *Target Nodes* list, select the target node you want to map.
```

Comment:
If I recall correctly, there's one more step after selecting sth like "Map these two nodes" button.

Reply:
Good catch, checked the UI and adopted as:
. Click *Map these two nodes*.